packages/hoppscotch-common/src/components/embeds/index.vue

```diff
@@ -1,105 +1,34 @@
-          blank
+  <AppPaneLayout layout-id="embed-primary" :is-embed="true">
+    <template #primary>
+      <EmbedsHeader :shared-request-u-r-l="sharedRequestURL" />
+      <div class="flex flex-col flex-1">
+        <EmbedsRequest
+          :model-tab="modelTab"
+          :shared-request-u-r-l="sharedRequestURL"
         />
-        <div class="flex">
-          <HoppButtonSecondary
-            :label="t('app.open_in_hoppscotch')"
-            :to="sharedRequestURL"
-            blank
+        <div class="flex flex-col flex-1">
+          <HttpRequestOptions
+            v-model="tab.document.request"
+            v-model:option-tab="selectedOptionTab"
+            :properties="properties"
+            :envs="tabRequestVariables"
           />
         </div>
       </div>
-    </header>
-    <div class="sticky top-0 z-10 flex-1">
-      <div
-        class="flex-none flex-shrink-0 p-4 bg-primary sm:flex sm:flex-shrink-0 sm:space-x-2"
-      >
-        <div
-          class="flex flex-1 overflow-hidden border divide-x rounded text-secondaryDark divide-divider min-w-[12rem] overflow-x-auto border-divider"
-        >
-          <span
-            class="flex items-center justify-center px-4 py-2 font-semibold transition rounded-l"
-          >
-            {{ tab.document.request.method }}
-          </span>
-          <div
-            class="flex items-center flex-1 flex-shrink-0 min-w-0 truncate rounded-r"
-          >
-            <SmartEnvInput
-              v-model="tab.document.request.endpoint"
-              :readonly="true"
-              :envs="tabRequestVariables"
-            />
-          </div>
-        </div>
-        <div class="flex mt-2 space-x-2 sm:mt-0">
-          <HoppButtonPrimary
-            id="send"
-            :title="`${t(
-              'action.send'
-            )} <kbd>${getSpecialKey()}</kbd><kbd>↩</kbd>`"
-            :label="`${!loading ? t('action.send') : t('action.cancel')}`"
-            class="flex-1 min-w-20"
-            outline
-            @click="!loading ? newSendRequest() : cancelRequest()"
-          />
-          <div class="flex">
-            <HoppButtonSecondary
-              :title="`${t(
-                'request.save'
-              )} <kbd>${getSpecialKey()}</kbd><kbd>S</kbd>`"
-              :label="t('request.save')"
-              filled
-              :icon="IconSave"
-              class="flex-1 rounded"
-              blank
-              outline
-              :to="sharedRequestURL"
-            />
-          </div>
-        </div>
-      </div>
-    </div>
-    <HttpRequestOptions
-      v-model="tab.document.request"
-      v-model:option-tab="selectedOptionTab"
-      :properties="properties"
-      :envs="tabRequestVariables"
-    />
-    <HttpResponse
-      v-if="tab.document.response"
-      :document="tab.document"
-      :is-embed="true"
-    />
-  </div>
+    </template>
+    <template #secondary>
+      <HttpResponse :document="tab.document" :is-embed="true" />
```

Comment:
Is it intentional not to conditionally render the response based on `tab.document.response` anymore?

Reply:
since we use Panes now hiding the response makes the pane empty.. which doesn't look good 😅  open for suggestion

![image](https://github.com/user-attachments/assets/d6fe10ce-2dba-432f-8e24-50596acc44b3)

Reply:
Gotcha, that makes sense.